reporec/github.py

```diff
@@ -26,7 +27,16 @@ def get_api(*paths):
     """
     path = _base + "/".join(paths)
     r = requests.get(path, headers=_build_header())
-    r.raise_for_status()
+    try:
+        r.raise_for_status()
+    except HTTPError as e:
+        if e.response.status_code == 403:
+            raise RuntimeError("A 403 Forbidden error occurred. You may need "
+                               "to get a GitHub Personal Access Token with "
+                               "full access to repo and export it as "
+                               "GITHUB_TOKEN.")
+        else:
+            raise
```

Comment:
Not sure what happened to my comment, but it may be more elegant here to explicitly check for the status code. So 200 would pass, 403 would raise a specific error, otherwise, we could call the `r.raise_for_status()`.

Should simplify this nested logic here.